### PR TITLE
Added yamls for bucket granular sync policy

### PR DESCRIPTION
**Pull request by TejasC88**
Signed-off-by: Tejas Chandramouli <tchandra@redhat.com>

Pass log : http://magna002.ceph.redhat.com:/ceph-qe-logs/tejas/cephci-run-GWGDG4/

### CODE REVIEW

**Review by anrao19 — APPROVED** [review]

**Review by viduship — APPROVED** [review]

File: suites/pacific/rgw/tier-1-extn_rgw_multisite-secondary-to-primary.yaml [hunk 1]
```diff
@@ -352,8 +352,34 @@ tests:
       polarion-id: CEPH-83575360
       module: sanity_rgw_multisite.py
       clusters:
-        ceph-sec:
+        ceph-pri:
```
Comment (ckulal) [inline suites/pacific/rgw/tier-1-extn_rgw_multisite-secondary-to-primary.yaml:355]: Looks good, But IMO, this suite file contains all the test-cases that are executed in secondary site and validated from primary site.

**Review by udaysk23 — APPROVED** [review]

File: suites/pacific/rgw/tier-1-extn_rgw_multisite-secondary-to-primary.yaml (hunk 1 above)
Comment (TejasC88) [reply suites/pacific/rgw/tier-1-extn_rgw_multisite-secondary-to-primary.yaml:355]: @ckulal that is correct , but does it make sense to create a new tier-1 suite yaml for just 2 test cases?

Comment (ckulal) [reply suites/pacific/rgw/tier-1-extn_rgw_multisite-secondary-to-primary.yaml:355]: IMO, we can create new suites or else we can change suite name accordingly,
if we make the suite name change please make sure to change the names in respective metadata file also.

File: pipeline/metadata/5.3.yaml [hunk 2]
```diff
@@ -639,6 +639,22 @@
...
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - sanity
+    - tier-2
+    - openstack
+    - ibmc
+    - rgw
+    - stage-2
```
Comment (anrao19) [inline pipeline/metadata/5.3.yaml:656]: Looks like for tag "sanity, tier-2, stage-2" we already have 5 suites, so might need to change the stage.
And also if this suite need to be executed as a part of 6.0 testing then changes required in 6.0 metadata file

**Review by anrao19 — APPROVED** [review]

**Review by ckulal — APPROVED** [review]

**Pull request merged**